parquet/src/main/java/org/apache/iceberg/parquet/Parquet.java

```diff
@@ -280,6 +283,90 @@ WriteBuilder withWriterVersion(WriterVersion version) {
             metricsConfig);
       }
     }
+
+    private static class Context {
```

Comment:
Nit:  I have the similar [comment](https://github.com/apache/iceberg/pull/2851/files#r675352093) for this class `Context`.

Reply:
Will update.

Reply:
Fixed.